kubernetes/cray-service/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [11.1.0]
```

Comment:
This is 100% backwards compatible, so I think a patch would be fine: 11.0.1

Reply:
I have updated this to version 11.0.1